seismicpro/src/seismic_batch.py

```diff
@@ -257,15 +296,16 @@ def copy_meta(self, from_comp, to_comp):
 
         for fr_comp, t_comp in zip(from_comp, to_comp):
             if fr_comp not in self.meta:
-                raise ValueError('{} does not exist.'.format(fr_comp))
+                raise ValueError('Meta of the component {} does not exist.'.format(fr_comp))
 
             if fr_comp == t_comp:
                 continue
 
-            if self.meta[t_comp]:
-                warnings.warn("Meta of component {} is not empty and".format(t_comp) + \
-                              " will be replaced by the meta from component {}.".format(fr_comp),
+            if t_comp in self.meta and self.meta[t_comp] != dict():
```

Comment:
```suggestion
            if self.meta.get(t_comp, None):
```

empty dict evaluates to false

Reply:
please, decide with @nikita-klsh which approach is better..